projects/driver_controls/test/test_ads1015.c

```diff
@@ -11,9 +11,10 @@
 #include "unity.h"
 
 static Ads1015Storage s_storage;
+static bool callback_called[4];
```

Comment:
`s_`

Also probably good idea to add a comment explaining why this has to be static